### PR TITLE
fix: [CDS-59590]: removed featureflag AZURE_WEBAPP_NG_JENKINS_ARTIFACTS

### PR DESCRIPTION
**Pull request by sahithikolichala**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sahithikolichala** [comment]
retrigger titlecheck

**Comment by sahithikolichala** [comment]
retrigger titlecheck

File: src/modules/70-pipeline/components/ArtifactsSelection/ServiceV2ArtifactsSelection.tsx [hunk 1]
```diff
@@ -181,7 +180,6 @@ export default function ServiceV2ArtifactsSelection({
       [ServiceDeploymentType.AzureWebApp, ServiceDeploymentType.TAS].includes(
         deploymentType as ServiceDeploymentType
       ) &&
-      AZURE_WEBAPP_NG_JENKINS_ARTIFACTS &&
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ServiceV2ArtifactsSelection.tsx:184]: Now `ENABLED_ARTIFACT_TYPES.Jenkins` need to be added in https://github.com/harness/harness-core-ui/blob/develop/src/modules/70-pipeline/components/ArtifactsSelection/ArtifactHelper.ts#L183.

Comment (pankajsingh-harness) [reply src/modules/70-pipeline/components/ArtifactsSelection/ServiceV2ArtifactsSelection.tsx:184]: Also the FF need to be removed from featureFlag file.
https://github.com/harness/harness-core-ui/blob/develop/src/modules/10-common/featureFlags.ts#LL91C1-L91C1

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactsSelection.tsx [hunk 2]
```diff
@@ -124,7 +123,6 @@ export default function ArtifactsSelection({
       [ServiceDeploymentType.AzureWebApp, ServiceDeploymentType.TAS].includes(
         deploymentType as ServiceDeploymentType
       ) &&
-      AZURE_WEBAPP_NG_JENKINS_ARTIFACTS &&
```
Comment (pankajsingh-harness) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactsSelection.tsx:127]: Now ENABLED_ARTIFACT_TYPES.Jenkins need to be added in https://github.com/harness/harness-core-ui/blob/develop/src/modules/70-pipeline/components/ArtifactsSelection/ArtifactHelper.ts#L183.

**Review by pankajsingh-harness — CHANGES_REQUESTED** [review]

**Pull request closed without merging**